simpa/utils/libraries/tissue_library.py

```diff
@@ -12,27 +12,39 @@
 from simpa.utils.calculate import randomize_uniform
 from simpa.utils.libraries.spectrum_library import AbsorptionSpectrumLibrary
 
+import torch
+
 
 class TissueLibrary(object):
     """
-    TODO
+    A library, returning molecular compositions for various typical tissue segmentations.
     """
 
-    def get_blood_volume_fractions(self, total_blood_volume_fraction=1e-10, oxygenation=1e-10):
+    def get_blood_volume_fractions(self, oxygenation: float | int | torch.Tensor = 1e-10,
+                                   blood_volume_fraction: float | int | torch.Tensor = 1e-10)\
+            -> list[int | float | torch.Tensor]:
```

Comment:
I would still go for typing.Union since the | operator does not work for python3.9